scipy/signal/signaltools.py

```diff
@@ -470,12 +471,10 @@ def convolve(in1, in2, mode='full'):
     if volume.ndim == 1 and kernel.ndim == 1 and volume.size >= kernel.size:
         return np.convolve(volume, kernel, mode)
 
-    slice_obj = [slice(None, None, -1)] * len(kernel.shape)
+    # Reverse in all dimensions
+    reverse = [slice(None, None, -1)] * kernel.ndim
 
-    if np.iscomplexobj(kernel):
-        return correlate(volume, kernel[slice_obj].conj(), mode)
-    else:
-        return correlate(volume, kernel[slice_obj], mode)
+    return correlate(volume, kernel[reverse].conj(), mode)
```

Comment:
Can you add a comment that using `.conj()` is faster than checking `iscomplexobj` even if it's not necessary to do it (for real inputs), like you mentioned in the comments of this PR?